src/app/components/playlist/playlist.component.ts

```diff
@@ -104,7 +104,7 @@ export class PlaylistComponent implements OnInit {
 
   search(q) {
     this.searchLoading = true;
-    this._spotify.search(q, "track,playlist", {limit: 50, market: 'from_token'})
+    this._spotify.search(q, "track,playlist", {limit: 30, market: 'from_token'})
```

Comment:
Why are we changing the search limit? This is outside the scope of this PR unless you want to add an infinite scroll on the search section. Which I would be 100% okay with.